### PR TITLE
[review] Increase the graphql knowledge

### PR DESCRIPTION
**Pull request by oliviertassinari**
Step, by step

### CODE REVIEW

File: src/server/graphql/ScreenshotDiffType.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+} from 'graphql';
+import ScreenshotDiff from 'server/models/ScreenshotDiff';
+
+export const resolve = (source, args) => {
+  return ScreenshotDiff
+    .query()
+    .where({
+      build_id: args.id,
+    })
+    .then((a) => {
```
Comment (neoziro) [inline src/server/graphql/ScreenshotDiffType.js]: ?

Comment (oliviertassinari) [inline src/server/graphql/ScreenshotDiffType.js]: It's a WIP, I have submitted the PR before taking the train. I need filter with the build id.

Comment (neoziro) [inline src/server/graphql/ScreenshotDiffType.js]: OK

**Comment by oliviertassinari** [comment]
I think that we have what we need now.
capture d ecran 2017-01-14 a 22 34 56

**Pull request merged**